src/main/java/org/openlmis/requisition/web/RequisitionController.java

```diff
@@ -161,6 +167,16 @@ public Collection<RequisitionPeriodDto> getProcessingPeriodIds(
         programId, facilityId, emergency
     );
 
+    if (unfinished) {
+      SupportedProgramDto program = facilitySupportsProgramHelper.getSupportedProgram(facilityId,
+              programId);
+      LocalDate programStartDate = program.getSupportStartDate();
+      periods = periods.stream()
+              .filter(p -> p.getStartDate().isEqual(programStartDate)
+                      || p.getStartDate().isAfter(programStartDate))
+              .collect(Collectors.toList());
```

Comment:
You can probably change isEqual and isAfter calls to single (p -> !p.getStartDate().isBefore(programStartDate))

Reply:
I made changes as suggested